src/cdk-experimental/popover-edit/edit-ref.ts

```diff
@@ -28,7 +28,8 @@ export class EditRef<FormValue> implements OnDestroy {
   readonly blurred: Observable<void> = this._blurredSubject.asObservable();
 
   /** The value to set the form back to on revert. */
-  private _revertFormValue: FormValue;
+  // TODO(issue/13329): Attempt to remove "!".
```

Comment:
I have a similar concern to @Splaktar here that these may start showing up in docs and also that they'll hang around forever like on the framework repo. I tried doing a big bang migration of this before (https://github.com/angular/components/pull/13338), but it was too hard to land in g3. Me and @devversion were talking that we could roll this out incrementally on a per-module basis by enabling it in each of the individual `tsconfig.json`. If there aren't any presubmit issues for a module, we can just fix the issues and land them, otherwise we can add these assertions only to that specific case.

Reply:
I can guarantee you that this change will land in g3, because it creates no runtime changes, and actually this was generated by a passing internal change. Using better fixes like `?` and `= null` do create observable changes at runtime which make it much harder to land.

I just want to warn you that by doing slower you will be more tempted to make better fixes, which will be harder to land because of course someone depends on the old behavior, and likely run out of steam and still not have the flag one year later.

I had basically a day to clean up some special casing around compiler flags in g3 internal compilations, which lead me to this. But now my cleanup day is over, so you can decide what to do with this PR.

I will ping #19339 one year from now :)

Reply:
On the subject of docs:  I believe // comments don't show up in TS docs, at least how TS perceives them.

You can experiment by hovering in your IDE

```
interface x {
  /** foo */
  // bar
  a: string;
}
```